site/content/en/docs/Reference/gameserver.md

```diff
@@ -96,7 +97,118 @@ spec:
+    # Defaults to 3. Minimum value is 1
+    failureThreshold: 3
+  # Parameters for game server sidecar
+  sdkServer:
+    # sdkServer log level parameter has three options:
+    #  - "Info" (default) The SDK server will output all messages except for debug messages
+    #  - "Debug" The SDK server will output all messages including debug messages
+    #  - "Error" The SDK server will only output error messages
+    logLevel: Info
+    # grpcPort and httpPort control what ports the sdkserver listens on.
+    # Starting with Agones 1.2 the default grpcPort is 9357 and the default
+    # httpPort is 9358. In earlier releases, the defaults were 59357 and 59358
+    # respectively but as these were in the ephemeral port range they could
+    # conflict with other TCP connections.
+    grpcPort: 9357
+    httpPort: 9358
+  # [Stage:Alpha]
+  # [FeatureFlag:PlayerTracking]
+  # Players provides the configuration for player tracking features.
+  # Commented out since Alpha, and disabled by default
+  # players:
+  #   # set this GameServer's initial player capacity
+  #   initialCapacity: 10
+  #
+  # [Stage:Alpha]
+  # [FeatureFlag:CountsAndLists]
+  # Counts and Lists provides the configuration for generic (player, room, session, etc.) tracking features.
+  # Commented out since Alpha, and disabled by default
+  # counters: # counters are int64 counters that can be incremented and decremented by set amounts. Keys must be declared at GameServer creation time.
+  #   games: # arbitrary key.
+  #     count: 1 # initial value.
+  #     capacity: 100 # (Optional) Maximum value for the counter. 0 is max(int64).
+  #   sessions:
+  #     count: 1
+  # lists: # lists are lists of values stored against this GameServer that can be added and deleted from. Keys must be declared at GameServer creation time.
+  #   players: # an empty list, with a capacity set to 10.
+  #     capacity: 10 # capacity value, defaults to 1000.
+  #   rooms:
+  #     capacity: 333
+  #     values: # initial set of values in a list.
+  #       - room1
+  #       - room2
+  #       - room3
+  #  
+  # Pod template configuration
+  # {{< k8s-api-version href="#podtemplate-v1-core" >}}
+  template:
+    # pod metadata. Name & Namespace is overwritten
+    metadata:
+      labels:
+        myspeciallabel: myspecialvalue
+    # Pod Specification
+    spec:
+      containers:
+      - name: simple-game-server
+        image:  {{< example-image >}}
+        imagePullPolicy: Always
+      # nodeSelector is a label that can be used to tell Kubernetes which host
+      # OS to use. For Windows game servers uncomment the nodeSelector
+      # definition below.
+      # Details: https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-containers/#ensuring-os-specific-workloads-land-on-the-appropriate-container-host
+      # nodeSelector:
+      #   kubernetes.io/os: windows
+```
+{{% /feature %}}
```

Comment:
Same as the markdown files - need each of the fields described below please (line 219 onwards)

Reply:
I think you intended for the counters and lists fields to be included within the `spec` field?